libraries/libfc/include/fc/string.hpp

```diff
@@ -20,4 +21,22 @@ namespace fc
   class variant_object;
   std::string format_string( const std::string&, const variant_object&, bool minimize = false );
   std::string trim( const std::string& );
+
+  /**
+   * Convert '\t', '\r', '\n', '\\' and '"'  to "\t\r\n\\\"" if escape_control_chars == true
```

Comment:
Should "\t\r\n\\\"" be "\\t\\r\\n\\\\"?

Reply:
Github automatically re-formatted two `\\t` into one `\t` above. In the test, `\t` becomes `\\t`.

Reply:
Ah, so you think the comment should be:
```
Convert '\t', '\r', '\n', '\\' and '"'  to "\\t\\r\\n\\\\\"" if escape_ctrl == on
```
?

Reply:
I guess I read the comment as saying that is what the output would look like, not what its internal representation in the string would be.